client/my-sites/checkout/checkout-thank-you/redesign-v2/sections/footer/DomainTransferToAnyUserFooter.tsx

```diff
@@ -26,6 +31,10 @@ const DomainTransferToAnyUserFooter = () => {
 					onClick={ null }
 				/>
 			</div>
+			<ProfessionalEmail
```

Comment:
I'm thinking to break this down into an upsell section instead of directly including it in the footer.